taraxa/state/chain_config/chain_config.go

```diff
@@ -7,9 +7,19 @@ import (
 	dpos "github.com/Taraxa-project/taraxa-evm/taraxa/state/dpos/precompiled"
 )
 
+type BlockRewardsOpts struct {
```

Comment:
ok maybe my previous comment is not right. This is already in master and it shows as new change too. I probably broke your PR by doing force push to previous versions ?